strapi/strapi
# [i18n] allows to switch default locale

### What does it do?

Add advanced settings in ModalEdit form of i18n settings in order to allow for changing the default locale

### How to test it?

- Log in the app with I18N plugin loaded
- Go to the settings menu item in the left sidebar
- Click "Internationalisation" in the left sub-sidebar
- Make sure to have at least two locales
- Edit the default locale and go to the advanced tab
- **The checkbox should be checked and disabled**
- Close this modal
- Edit a locale that is NOT the default one
- Go to the advanced tab
- Click on the "Set default locale" box
- Save
- **The modal should close, show a toast and refresh the list with the good default locale**

### Screenshots

![2021-02-25 13 42 21](https://user-images.githubusercontent.com/3874873/109155999-984a9800-7770-11eb-86df-9170322848c3.gif)
